darwin/cli_functions.py

```diff
@@ -698,7 +698,7 @@ def file_upload_callback(file_name, file_total_bytes, file_bytes_sent):
         already_existing_items = []
         other_skipped_items = []
         for item in upload_manager.blocked_items:
-            if item.reason.upper() == "ALREADY_EXISTS":
+            if (item.reason is not None) and (item.reason.upper() == "ALREADY_EXISTS"):
```

Comment:
No, this was static type analyser fixes - I'm not religious about them, but they do reveal issues sometimes, and did to an extent this time, so I'm basically running `mypy` against files I change, in an effort to slowly improve things without a slew of tech debt tickets.

I always make it the final commit so that if anyone wants to see the focus of the work, they can diff the commits instead.